api/users/serializers.py

```diff
@@ -102,6 +102,8 @@ def update(self, instance, validated_data):
                 setattr(instance, attr, value)
         try:
             instance.save()
+        except ValidationError as e:
+            raise InvalidModelValueError(detail=e.args)
```

Comment:
@TomBaxter How about `e.args[0]`? That is a typical way to get the error message for a given exception.